perception/object_merger/src/object_association_merger/node.cpp

```diff
@@ -34,9 +34,9 @@ namespace
 bool isUnknownObjectOverlapped(
   const autoware_auto_perception_msgs::msg::DetectedObject & unknown_object,
   const autoware_auto_perception_msgs::msg::DetectedObject & known_object,
-  const double precision_threshold, const double recall_threshold)
+  const double precision_threshold, const double recall_threshold, double distance_threshold)
```

Comment:
This function should originally only require precision_threshold and recall_threshold. This is because distance_threshold is a parameter to reduce computational cost.
What about setting a large enough value in the default argument?